src/notation/view/widgets/pagesettings.cpp

```diff
@@ -268,6 +268,8 @@ void PageSettings::on_resetPageStyleButton_clicked()
     }
 
     updateValues();
+    globalContext()->currentNotation()->undoStack()->commitChanges();
```

Comment:
I must admit that I am not entirely sure why this is required, but it does not fully work without it.

Reply:
A bit of context about what this line does:

Each thing that the user does needs to be undoable. So before a change, we call `undoStack()->prepareChanges()`, to start recording changes. After the changes, we call either `rollbackChanges()` or `commitChanges()`, to revert the changes, or to "commit" them. A block of committed changes can be undone/redone.

In the case of the PageSettings dialog, we call `prepareChanges` when the user opens the dialog, `rollbackChanges` when the user clicks Cancel and `commitChanges` when the user clicks OK. So it is not desired that we commit the changes earlier than that, so we need another solution.

I looked why the score does update correctly after _changing_ a style value and not when _resetting_. The difference seems to be in the NotationStyle class:
<img width="547" alt="Scherm­afbeelding 2022-12-26 om 15 53 52" src="https://user-images.githubusercontent.com/48658420/209560918-d9b5b7a7-69ae-4d4a-aaca-8b20d21bf3df.png">
The `resetStyleValue` method seems to be missing a call to `score()->update()`.